src/Libraries/Microsoft.Extensions.Http.AutoClient/AutoClientUtilities.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.Http.AutoClient;
+
+
+/// <summary>
```

Comment:
```suggestion
namespace Microsoft.Extensions.Http.AutoClient;

/// <summary>
```
NIT: Extra new line causing PR build failure.

Reply:
Feel free to edit the files directly in GH, I won't mind